package.json

```diff
@@ -25,6 +25,6 @@
     "body-parser": "^1.19.0",
     "ejs": "^3.0.1",
     "express": "^4.17.1",
-    "transbank-sdk": "^1.0.0"
+    "tbk_sdk": "file:../tbk_sdk"
```

Comment:
We are going to upload the package to npm as transbank-sdk when it's ready, so this change is not needed :)